### PR TITLE
Test cypress integration

### PR DESCRIPTION
**Pull request by tbille**
# THIS IS A TEST PR DON'T MERGE!!!

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,3 +1,6 @@
 PORT=8004
 SECRET_KEY=local_development_fake_key
-WTF_CSRF_SECRET_KEY=this_is_a_random_string
\ No newline at end of file
+WTF_CSRF_SECRET_KEY=this_is_a_random_string
+SNAPCRAFT_IO_API=https://api.staging.snapcraft.io/api/v1/
```
Comment (tbille) [inline .env]: This won't be merged it's just to test for now

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,20 @@
 sudo: required
+language: node_js
```
Comment (tbille) [inline .travis.yml]: This comes from the basic configuration for Cypress: https://docs.cypress.io/guides/guides/continuous-integration.html#Travis

File: .travis.yml [hunk 3]
```diff
@@ -1,7 +1,20 @@
...
+install:
+  - npm install
 
 services:
   - docker
 
 script:
   - ./run test
+  - ./run &
+  - wait 10
```
Comment (tbille) [inline .travis.yml]: I just want to let time to the run script to start (should find a better way)

File: cypress.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+  "chromeWebSecurity": false
```
Comment (tbille) [inline cypress.json]: This is needed for redirections from login SSO in HTTPS to our local s.io (not in HTTPS)

File: cypress/fixtures/example.json [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (tbille) [inline cypress/fixtures/example.json]: Generated file: to be removed

File: cypress/fixtures/profile.json [hunk 6]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (tbille) [inline cypress/fixtures/profile.json]: Generated file: to be removed

File: cypress/fixtures/users.json [hunk 7]
```diff
@@ -0,0 +1,232 @@
+[
```
Comment (tbille) [inline cypress/fixtures/users.json]: Generated file: to be removed

File: cypress/plugins/index.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
+// ***********************************************************
```
Comment (tbille) [inline cypress/plugins/index.js]: Generated file: to be removed

File: cypress/support/commands.js [hunk 9]
```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```
Comment (tbille) [inline cypress/support/commands.js]: Generated file: to be removed

File: cypress/support/index.js [hunk 10]
```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
```
Comment (tbille) [inline cypress/support/index.js]: Generated file: to be removed

File: .travis.yml [hunk 11]
```diff
@@ -1,7 +1,18 @@
...
+    - ~/.npm
+    - node_modules
+install:
+  - npm install
 
 services:
   - docker
 
 script:
-  - ./run test
```
Comment (tbille) [inline .travis.yml]: I removed this line, because the API calls are hardcoded, and I want to use staging to run cypress (for now)

**Comment by Lukewh** [comment]
Is this PR still relevant? Is the test complete? If not can we move it to a feature branch to clean up the PR list pleeeease?

**Comment by tbille** [comment]
This was a test PR. I am closing it.

**Pull request closed without merging**